subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -1844,11 +1844,14 @@ int lwm2m_write_handler(struct lwm2m_engine_obj_inst *obj_inst,
 	data_len = res->data_len;
 
 	/* setup data_ptr/data_len for OPAQUE when it has none setup */
+/* FIXME: */
```

Comment:
FYI: these 2 FIXME's will be resolved in a future patch dealing with OPAQUE handling.  The current uses are direct writing of firmware to the 5/0/0 (package) resource and bootstrap process for writing DTLS secret.